### PR TITLE
Improved UX based on user feedback

### PR DESCRIPTION
**Pull request by nicocartesi**
Based on multiple feedback sessions, it was pointed out that the landing page needs to direct readers to the required section. Furthermore, for new readers, it is not clear what Cartesi exactly entails.

This PR aims to:

- Provide a short description of Cartesi for a better understanding of Cartesi Framework
- Provide a shortcut to the required information, especially to the hands-on part on building DApps

Screenshot 2023-07-30 at 21 56 15

### CODE REVIEW

**Comment by nicocartesi** [comment]
Hi @dimitarangelkov , can you please review this PR from the UX perspective? Thanks!

**Comment by felipeargento** [comment]
Hello!

To be honest, I didn't like this description very much. The text, as it stands, appears to be repetitive in some places, and somewhat confusing in others. There are also some grammar mistakes and inconsistencies.  For instance, "Cartesi Rollups ecosystem" should be "The Cartesi Rollups ecosystem" to be grammatically correct. In addition, the capitalization of "DApps" is inconsistent, appearing as "Dapps" later in the text.

I'd consider reworking the description around the three phrases found on the cartesi.io website. They are concise, easy to understand, and accurately represent what Cartesi offers:

1) "Application-specific rollups with a Linux runtime."

2) "Cartesi gives your DApp its own rollup with a dedicated CPU, bringing significant gains in computational scalability without compromising on decentralization, security and censorship resistance."

3) "With the Cartesi Virtual Machine, DApp developers move beyond the EVM and gain access to the full suite of code libraries, programming languages, and open-source tooling they're accustomed to."

**Comment by dimitarangelkov** [comment]
@nicocartesi I'll fine tune the two buttons styles, to match the branding guide lines etc. Other than that, the text under the main headline is a bit too long for that part of the screen. Nobody wants to read long sentences.

File: src/components/Hello.js [hunk 1]
```diff
@@ -5,13 +5,40 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
...
+    margin: "5px",
+    fontWeight: "bold", 
+  };
+
+  const buttonContainerStyle = {
+    display: "flex",
+    justifyContent: "center",
+    gap: "10px",
+    marginTop: "20px"
+  };
```
Comment (dimitarangelkov) [inline src/components/Hello.js:24]:
```
const buttonStyle = {
    padding: "0.75rem 0.75rem",
    textAlign: "center",
    textDecoration: "none",
    backgroundColor: "#1B1A1E",
    color: "#fff",
    borderRadius: "0.25rem",
    fontWeight: "bold",
    textTransform: "uppercase",
    width: "100%",
  };

  const buttonContainerStyle = {
    display: "flex",
    justifyContent: "center",
    gap: "0.75rem",
    marginTop: "1.5rem",
  };
```

Comment (dimitarangelkov) [reply src/components/Hello.js:24]: @nicocartesi here are the updated styles

**Review by dimitarangelkov — CHANGES_REQUESTED** [review]
Added a comment with the updated styles

**Comment by nicocartesi** [comment]
Thanks @felipeargento and @dimitarangelkov for the feedback! I have amended the text in line with @felipeargento 's suggestion but shortened it a bit as per @dimitarangelkov 's suggestion. Added the button styles too:
Screenshot 2023-08-04 at 17 39 06

Please let me know if this version is good to go. Thanks!

**Review by dimitarangelkov — APPROVED** [review]
lgtm!

**Pull request merged**